data/ships.txt

```diff
@@ -724,6 +724,9 @@ ship "Carrier"
 		"Meteor Missile" 140
 		"Heavy Laser Turret" 2
 		"Heavy Anti-Missile Turret" 2
+
+		"Laser Rifle" 40
+		"Fragmentation Grenades" 40
```

Comment:
See the Marauder ships file. H2H weapons don't get their own "block" like this separated from the rest of the outfits. They go at the bottom of the center block.

Reply:
Sure thing. Done in b087f88.